package.json

```diff
@@ -55,9 +49,15 @@
   },
   "devDependencies": {
     "@octokit/rest": "^18.12.0",
+    "@types/classnames": "^2.3.1",
     "@types/eslint": "^8.4.1",
     "@types/glob": "^7.2.0",
+    "@types/lodash.throttle": "^4.1.7",
+    "@types/marked": "^2.0.0",
+    "@types/mdx-js__react": "^1.5.5",
     "@types/node": "^14.18.16",
+    "@types/prettier": "^2.6.0",
+    "@types/puppeteer": "^5.4.6",
     "@types/react": "^17.0.44",
     "@typescript-eslint/eslint-plugin": "^5.22.0",
     "@typescript-eslint/parser": "^5.22.0",
```

Comment:
Who is using all these Typescript dependencies 🤔  Afaik we don't have any typescript in the yarn scope of the monorepo (projects/)?

Reply:
Ahh wait I'm in the docs here 😅